### ISSUE TITLE
Cannot read property 'register' of undefined

### ISSUE THREAD

**Issue by Creatium** [Makanz/chartjs-plugin-trendline#54]
Hi,

Trying to use this plugin with Chart JS 3.2. Anything above Chart JS 2.9.4 gives me this error:

```
Uncaught TypeError: Cannot read property 'register' of undefined
    at chartjs-plugin-trendline.min.js:formatted:93
```

Trendline is inlined after chartjs. Chart is being created after an AJAX call for data.

Calling "window.Chart.plugins" on document ready gives "undefined". Is this something that changed with version 3 of Chart JS and needs an update on Trendline side?

**Comment by kuzudevelopers** [comment]
I can also reproduce this error using inline scripts from cdnjs.com: anything over ChartJS 2.9.3 (latest available version of v2 on cdnjs) causes this error.

**Comment by Makanz** [comment]
Thanks for the report. I will take a look at it.

**Comment by jremi** [comment]
@Makanz i'm going to try to review this also, but do you happen to have any ETA on when you may have Chart JS 3.x support so we can use this plugin?

Thx 🙏

**Comment by husain3012** [comment]
@Makanz i would like to work on this issue, i figured out the issue causing this bug

**Comment by Makanz** [comment]
I have now merged the PR and created a new release v1.0.0. Its also published to npm.
